tests/unit/routers/test_fact.py

```
@@ -154,6 +154,27 @@ def test_get_fact_fact_returned_for_html(
+    )
+    mocker.patch("application.routers.fact.get_dataset_query", return_value=None)
+    request = MagicMock()
+    try:
+        get_fact(
+            request=request,
+            path_params=path_params,
+            query_filters=query_params,
+            extension=None,
+            session=MagicMock(),
+        )
+        assert False, "Expected HTTPException to be raised"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)